paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -72,6 +75,12 @@ internal class CustomerApiRepository @Inject constructor(
                     ),
                 ).onFailure {
                     logger.error("Failed to retrieve payment methods.", it)
+                    val stripeException = StripeException.create(it)
+                    errorReporter.report(
```

Comment:
Might be worth adding an extension function to extract these fields from a `StripeException`, I'm guessing we'll have a lot of code that looks like this otherwise.